Haneke/Fetcher.swift

```diff
@@ -17,7 +17,7 @@ public class Fetcher<T : DataConvertible> {
         self.key = key
     }
     
-    public func fetch(failure fail: ((NSError?) -> ()), success succeed: (T.Result) -> ()) {}
+    public func fetch(failure fail: ((NSError?) -> ()), success succeed: @escaping (T.Result) -> ()) {}
```

Comment:
`failure` should have `@escaping` as well. This happens on a lot of places in this PR, specially on fails.